lib/2_uppercases_all_letters.php

```diff
@@ -7,9 +7,29 @@
 // Run `bin/check` from the command line to execute the automated tests.
 
 function uppercasesAllLetters($str) {
-  // Your code goes here.
+  return strtoupper($str);
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
+echo "\nTesting that 'test' returns 'TEST'.\n";
+if (uppercasesAllLetters("test") == "TEST") {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that 'I' returns 'I'.\n";
+if (uppercasesAllLetters("I") == "I") {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\n###################################################################\n";
+echo "Tests complete.\n";
```

Comment:
It would seem far more likely that if it was going to fail on anything, it was going to fail on non-ASCII characters, though I don't have the necessary knowledge to work out exactly how PHP ought to be deciding to "uppercase" things like non-Latin alphabets.  But at that point I'm really just testing the PHP built-in function, not my code.